src/main/java/cubicchunks/asm/mixin/core/client/MixinWorldClient.java

```diff
@@ -47,15 +47,12 @@
 	@Shadow public abstract boolean invalidateRegionAndSetBlock(BlockPos pos, IBlockState blockState);
 
 	@Override public void initCubicWorld() {
+		super.initCubicWorld();
```

Comment:
Does it actually work with mixin? This class is dynamically merged with WorldClient and Mixin may have probems with some edge cases.